packages/fiori-generator-shared/src/translations/fiori-generator-shared.i18n.json

```diff
@@ -8,5 +8,8 @@
     },
     "debug": {
         "loggingConfigured": "Logging has been configured at log level: {{logLevel}}"
+    },
+    "telemetry": {
+        "unknownOs": "Unknown"
```

Comment:
Is this correct? Should it be
```
"unknownOs": "UnknownOs"

Reply:
No this is the existing telemetry property used, check `UNKNOWN_OS` in internal repo. So I assume this needs to remain the same for consistency